uwblueprint/project-read-frontend
# Create session and class default views

### Notion ticket link



["All classes" table](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=718c1a8fc83c43f886d197d4a3245b7f)
[Class default view](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=32687d6b499c4d43bdf2947d8bdcb162)



### Implementation description

- Added tabs & classes table to sessions page, displaying list of families in their enrolled classes



### Steps to test

1. Go to http://localhost:3000/sessions, try using different sessions and different class tabs to see that everything is working as expected



### What should reviewers focus on?

- Good typescript & react conventions, correctness of data from the API

### Checklist
